frontend/src/employee-frontend/components/messages/MessageEditor.tsx

```diff
@@ -196,9 +216,27 @@ export default React.memo(function MessageEditor({
       draftContent?.recipientIds
     )
   )
+  const [filtersVisible, setFiltersVisible] = useState(false)
```

Comment:
Consider using `useBoolean` hook instead which gives you a toggle function for free, so you can remove the manually written `toggleFilters`